src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -14,8 +14,13 @@ public static PatchMap Generate(WorldGenerationSettings settings, Species defaul
 
         var map = new PatchMap();
 
+        // Generate random name for patch
+        var nameGenerator = SimulationParameters.Instance.GetPatchMapNameGenerator();
+        var name = nameGenerator.Next();
+
         // Predefined patches
-        var vents = new Patch("PATCH_PANGONIAN_VENTS", 0,
+        string localizedName = GetPatchLocalizedName(name, "VOLCANIC_VENT").ToString();
```

Comment:
Instead of using an intermediate variable for the localized name I think you should just put the call in the patch constructor.